.github/workflows/release-python-bindings.yml

```diff
@@ -96,26 +96,59 @@ jobs:
           name: wheels
           path: dist
 
+  build-wheels-aarch64:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3 # Required to mount the Github Workspace to a volume
+      - uses: addnab/docker-run-action@v3
+        with:
+          image: python:3.9.12-slim-bullseye
+          options: -v ${{ github.workspace }}:/root
+          run: |
+            cd root
+
+            apt-get update 
+            apt-get install -y git curl build-essential libudev-dev libusb-1.0-0-dev pkg-config clang
+            curl https://sh.rustup.rs -sSf | sh -s -- -y 
+            . "$HOME/.cargo/env" 
+
+            cd wallet/bindings/python 
+            pip install -r requirements-dev.txt 
+            pip install patchelf
+            pip install maturin
+            maturin build --out ../../../dist --release
```

Comment:
Will dist exist if you don't create it ?

Reply:
Yes, just tested https://github.com/Thoralf-M/wallet.rs/actions/runs/4280579030/jobs/7452529593